iot-auth-service/image.yaml

```diff
@@ -52,7 +52,7 @@ run:
       user: 185
       cmd:
           - "/bin/launch_java.sh"
-          - "/opt/iot-component.jar"
```

Comment:
All IoT components are named `iot-component.jar` since that is used by the `hono.*` modules.

Reply:
Yes, i need to make the module install per image